### ISSUE TITLE
Explanations

### ISSUE THREAD

**Issue by Marco-Congedo** [josephcbradley/Hurst.jl#2]
Hello,

  in your example code:
```
H = hurst_exponent(X, 1:19)
```

could you please tell me what the `1:19` argument does and how these values should be chosen?

Thanks

**Comment by josephcbradley** [comment]
The ```1:19``` argument is that range of lags ($\tau$) that will be used to calculate the Hurst exponent. The formula for the Hurst exponent can be written as:

$\ln \left(E\left[|X(t+\tau)-X(t)|\right]\right)= H\ln (\tau)+ C$

where $\tau$ is the lag, $H$ is the Hurst exponent, $X$ is your time series, and $C$ is a constant. We calculate the Hurst exponent in this package by calculating $E\left[|X(t+\tau)-X(t)|\right]$ (the expectation) as a function of $\tau$ for the values in your $\tau$ range (e.g. ```1:19```). We then take logs of the expectation and $\tau$ and do a linear fit. The coefficient on that linear fit is the Hurst exponent. 

It turns out that finding the best values for $\tau$ is tricky and I have been meaning update the package examples which are not ideal. ```tau_range = 30:250``` will probably be better than ```1:19``` (see [this paper](http://dx.doi.org/10.1016/j.chaos.2015.11.022)). The best case I know of (see [here](https://doi.org/10.1103/PhysRevE.95.042311)) is to calculate the ideal range per time series, which I plan to implement here soon. 

tl;dr: it's the range of time lags you need to specify to calculate the Hurst exponent. Use ```30:250``` if you're unsure.

Let me know if you have any more questions!

**Comment by Marco-Congedo** [comment]
OK, thank you. I guess that the best tau range depends on the data at hand. Looking forward to see further evolution of this package!

**Comment by Marco-Congedo** [comment]
By the way, how come i get slightly negative values of Hurst exponent sometimes?

**Comment by Marco-Congedo** [comment]
I am computing the Hurst exponent on pretty long time series (tens of thousands of samples). Should i send you an example of the data?

**Comment by josephcbradley** [comment]
Yes please - joseph.bradley@kcl.ac.uk

**Comment by Marco-Congedo** [comment]
OK, i'll do it this evening. Thanks
